pkg/ci-firewall/cli/worker/work.go

```diff
@@ -197,6 +203,8 @@ func NewWorkCmd(name, fullname string) *cobra.Command {
 	cmd.Flags().BoolVar(&o.redact, "redact", true, "if true, then injected envs and ip addresses are redacted from logs sent over queue. Default is true")
 	cmd.Flags().StringVar(&o.gitUser, "gituser", "", "The git user you want to configure for repo")
 	cmd.Flags().StringVar(&o.gitEmail, "gitemail", "", "The email of git user you want to configure on repo")
+	cmd.Flags().IntVar(&o.retryLoopCount, "retryloopcount", 3, "The number of times we retry command if it fails. Defaults to 3")
+	cmd.Flags().DurationVar(&o.retryLoopBackoff, "retryloopbackoff", 10 * time.Second, "The base amount of time to back off before retry on command execution due to execution errors. Defaults to 10 seconds. Actual backoff will be currentcount * backoff duration")
```

Comment:
You mean 3rd attempt will start after 30 sec of the second execution failure

Reply:
yes

Reply:
You can reduce it too if you want by maybe overriding default value with 5s so 2nd attempt will be after 10s and 3rd after 15s for eg

Reply:
Make sense